legend-sdlc-model/src/main/java/org/finos/legend/sdlc/domain/model/workflow/WorkflowJobStatus.java

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.domain.model.workflow;
+
+public enum WorkflowJobStatus
+{
+    PENDING, IN_PROGRESS, SUCCEEDED, FAILED, CANCELED, MANUAL, UNKNOWN
```

Comment:
I don't like "manual" as a status. To me that seems more like a description of the job, rather than the status of the job. I guess it's really a special kind of "pending": specifically, pending a manual trigger. It would be nice if we could find a better name for it that sounds more like a status and less like a type.

We may also want to consider adding some additional statuses. For example, "skipped". That's a job that wasn't and won't be run, but also wasn't canceled. And maybe we should have one for a job that hasn't yet been run and isn't currently scheduled to run (similar to "created").